src/Reflection/ReflectionParameter.php

```diff
@@ -228,6 +235,16 @@ private function parseDefaultValueNode() : void
         );
     }
 
+    private function findParentClassDeclaringConstant(string $constantName): string
+    {
+        $class = $this->function->getDeclaringClass();
+        do {
+            if ($class->hasConstant($constantName)) {
+                return $class->getName();
+            }
+        } while ($class = $class->getParentClass());
```

Comment:
Yes, I also think it is a dead code but I added it.
With it, it seems better to me 👍